Task2_Calculator/src/Calculator.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+
+public class Calculator {
+    public static void main(String[] args) {
+        String programm_name = "Calculator";
+        System.out.println(programm_name);
+        double number2 = 0.00; //Warning:(7, 26) Variable 'number2' initializer '0.00' is redundant
+        //       double number1 = 0.00;
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Enter the 1st number");
+        double number1 = scan.nextDouble();
+        //       if(number1.hasNextInt||number1.hasNextDouble()){// double cannot be dereferenced is
```

Comment:
I dont't catch what you are trying to check

Reply:
Task: 
If the user inputs a String instead of a number, output that input is wrong
I try to check whether input contains integer or not